arch/cortex-m4/src/scb.rs

```diff
@@ -33,6 +33,22 @@ const SCB_BASE: usize = 0xE000ED00;
 
 static mut SCB: *mut ScbRegisters = SCB_BASE as *mut ScbRegisters;
 
+/// Allow the core to go into deep sleep on WFI.
+///
+/// The specific definition of "deep sleep" is chip specific.
+pub unsafe fn set_sleepdeep() {
```

Comment:
I don't think so. Could change the comment, but this is literally just a utility function to set that bit in the Cortex-M4 register so the chip implementation doesn't deal with Cortex-M4-defined register maps.